src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/DropIndexCommand.java

```diff
@@ -1,24 +1,34 @@
 package io.stargate.sgv2.jsonapi.api.model.command.impl;
 
 import com.fasterxml.jackson.annotation.JsonTypeName;
-import io.stargate.sgv2.jsonapi.api.model.command.CollectionCommand;
+import io.stargate.sgv2.jsonapi.api.model.command.KeyspaceCommand;
 import io.stargate.sgv2.jsonapi.api.model.command.NoOptionsCommand;
+import jakarta.annotation.Nullable;
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Pattern;
 import jakarta.validation.constraints.Size;
+import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
-// TODO, hide table feature detail before it goes public,
-// https://github.com/stargate/data-api/pull/1360
-// @Schema(description = "Command that drops an index for a column.")
+@Schema(description = "Command that drops an index for a column.")
 @JsonTypeName("dropIndex")
 public record DropIndexCommand(
     @NotNull
         @Size(min = 1, max = 48)
         @Pattern(regexp = "[a-zA-Z][a-zA-Z0-9_]*")
```

Comment:
The RegExp pattern here is the CQL rules for unquoted identifiers. Do we allow the index name to be quoted ? Also is the length confirmed ?
as this is a drop index command, we would need to support any index name created in CQL. So do we need these restrictions ?